### ISSUE TITLE
Is this still correct?  opencpn/plugins has an Auto branch?

### ISSUE THREAD

**Issue by rgleason** [Rasbats/managed_plugins#54]
https://opencpn-manuals.github.io/main/AlternativeWorkflow/InstallConfigure/GitHub.html#_check_the_metadata_push

Also https://opencpn-manuals.github.io/main/AlternativeWorkflow/InstallConfigure/Catalog-Github-Integration.html
Much of this has to change, if so.

**Comment by rgleason** [comment]
Where is the auto branch supposed to be?

**Comment by rgleason** [comment]
```
**Wrong! The auto branch is in the fork of OpenCPN/plugins. Sorry if I did not make that clear.**

Mike 
```
OK, now I realize that 
I had created  an "auto" branch in my local "plugins" repository that matches opencpn/plugins/master, however I had not pushed it up to my github remote repository!  That is apparently why most of the plugins "failed" in git.  Click on the red X in this git link
https://github.com/rgleason/vdr_pi

However it appears that they did all build and land in the cloudsmith opencpn/vdr-prod repository which is very good! (I must have been tired last night and did not check.
https://cloudsmith.io/~opencpn/repos/vdr-prod/packages/?q=1.3.26

I will simply use my normal methods to make a PR to opencpn/plugins now, rather than increment the version, add and make a new commit, then add a new tag and then push --tags origin master.   I will then try to do this as expected for logbookkonni. now that I understand.

Closing
